imap_processing/tests/spice/test_kernels.py

```diff
@@ -1,33 +1,65 @@
 """Tests coverage for imap_processing/spice/kernels.py"""
 
+import numpy as np
 import pytest
 import spiceypy as spice
 from spiceypy.utils.exceptions import SpiceyError
 
-from imap_processing.spice import kernels
+from imap_processing.spice.kernels import (
+    _average_quaternions,
+    _create_rotation_matrix,
+    _get_et_times,
+    create_pointing_frame,
+    ensure_spice,
+)
+
+
+@pytest.fixture()
+def kernels(spice_test_data_path):
+    """List SPICE kernels."""
+    # TODO: remove this fixture after ensure_spice update.
+    kernels = [str(file) for file in spice_test_data_path.iterdir()]
+    return kernels
+
+
+@pytest.fixture()
+def et_times(kernels):
```

Comment:
This is kind of a weird combo of a fixture and test. It isn't really clear that it is a test by the name and in fact I don't think `pytest` will pick this up as a test. Can this just be a fixture and write a separate test for `_et_times`?